docs/_articles/en/getting-started/settings.md

```diff
@@ -1,10 +1,15 @@
 ---
-title: Edit Your Workspace Settings
+title: Settings
 lang: en
 ---
 
+## Workspace Settings
 To edit your workspace settings, select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS). For information about configuring your settings, see [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings) in the Visual Studio Code docs.
 
+## Extension Settings
+We use extension settings to provide access to beta features and other customizations.
```

Comment:
@ekapner - what do you think about linking to VS Code's doc page for settings?  Overkill?  this is the page i'm referring to - https://code.visualstudio.com/docs/getstarted/settings#_creating-user-and-workspace-settings